src/middlewares/auth.middleware.js

```diff
@@ -1,6 +1,32 @@
 const User = require('../models/user.models')
 const {response_400, response_500} = require('../utils/responseCodes.utils')
+const sendError=require('../utils/responseCodes.utils.js');
+const jwt = require('jsonwebtoken');
+const secretKey = "777";
```

Comment:
Don't define secreatKey here. Rather add it to .env and also add in .env.example for reference.